mqt/qusat/CMakeLists.txt

```
@@ -1,3 +1,3 @@
 pybind11_add_module(py${PROJECT_NAME} bindings.cpp)
-target_link_libraries(py${PROJECT_NAME} PUBLIC ${PROJECT_NAME} MQT::qfr_python pybind11_json)
+target_link_libraries(py${PROJECT_NAME} PRIVATE ${PROJECT_NAME} MQT::qfr_python pybind11_json)
```

```suggestion
target_link_libraries(py${PROJECT_NAME} PRIVATE ${PROJECT_NAME} MQT::qfr_python)
```
can be dropped since it's part of the `MQT::qfr_python` target